### ISSUE TITLE
Test clear_texture::clear_texture_2d_uncompressed fails on metal with METAL_DEVICE_WRAPPER_TYPE=1

### ISSUE THREAD

**Issue by cwfitzgerald** [gfx-rs/wgpu#2794]
**Description**

See title.

**Repro steps**

```
METAL_DEVICE_WRAPPER_TYPE=1 cargo nextest run --all --no-fail-fast
```

**Expected vs observed behavior**

Errors out with:

```
        FAIL [   0.099s]                  wgpu::wgpu-tests clear_texture::clear_texture_2d_uncompressed

--- STDOUT:                               wgpu::wgpu-tests clear_texture::clear_texture_2d_uncompressed ---

running 1 test
clearing texture with R8Unorm, dimension D1, size Extent3d { width: 64, height: 1, depth_or_array_layers: 1 }

--- STDERR:                               wgpu::wgpu-tests clear_texture::clear_texture_2d_uncompressed ---
2022-06-20 15:57:08.180 wgpu_tests-cde506ede13d2d24[59782:793409] Metal API Validation Enabled
validateRenderPassDescriptor:782: failed assertion `RenderPass Descriptor Validation
MTLRenderPassAttachmentDescriptor texture must not be MTLTextureType1D.
'

     Summary [   0.100s] 1 tests run: 0 passed, 1 failed, 74 skipped
```

**Comment by jinleili** [comment]
The WebGPU Spec says:
> A [GPUTextureView](https://gpuweb.github.io/gpuweb/#gputextureview) view is a renderable texture view if the following requirements are met:
view.[[[texture]]](https://gpuweb.github.io/gpuweb/#dom-gputextureview-texture-slot).[usage](https://gpuweb.github.io/gpuweb/#dom-gputexture-usage) must contain [RENDER_ATTACHMENT](https://gpuweb.github.io/gpuweb/#dom-gputextureusage-render_attachment).
descriptor.[dimension](https://gpuweb.github.io/gpuweb/#dom-gputextureviewdescriptor-dimension) must be ["2d"](https://gpuweb.github.io/gpuweb/#dom-gputextureviewdimension-2d).

So this fail maybe more of an alignment issue with the spec?

**Comment by cwfitzgerald** [comment]
Closed by #2818
